sdk/jobs/pipelines/1b_pipeline_with_python_function_components/pipeline_with_python_function_components.ipynb

```diff
@@ -122,8 +122,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# train_component_func = ml_client.components.create_or_update(train_component_func)\n",
-    "print(train_component_func)"
+    "# train_model = ml_client.components.create_or_update(train_model)\n",
```

Comment:
no need to have the create section in this sample

Reply:
removed